### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by realisticattorney**
Milestones were:

- App should render the Display and ButtonPanel component
- Implement the Display component
- Implement the ButtonPanel component
- Implement the Button component

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const result = '0';
```
Comment (addod19) [inline src/components/App.js:5]: [Optional]
- Kindly remove this

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const App = () => (
+  <div>
+    <Display result={result} />
+    <ButtonPanel />
+  </div>
```
Comment (addod19) [inline src/components/App.js:10]: Kindly wrap the children's components with the React.Fragment short syntax `<> </>`

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @realisticattorney, congratulations on this milestone :smile: The component directory created and contains the following component; "App", "Button", "Display", and the "ButtonPanel". Let us kindly work on and fix the App component. KIndly check the inline comments below:

Once this is resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi  Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by bellom — APPROVED** [review]
## Hi @realisticattorney  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**